kotlin/samples/todo-android/app/src/main/java/com/squareup/sample/mainactivity/TodoListsLayoutRunner.kt

```diff
@@ -19,27 +19,33 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
+import com.squareup.sample.todo.MasterDetailConfig.Master
 import com.squareup.sample.todo.R
 import com.squareup.sample.todo.TodoList
-import com.squareup.sample.todo.TodoListsRendering
+import com.squareup.sample.todo.TodoListsScreen
 import com.squareup.workflow.ui.LayoutRunner
 import com.squareup.workflow.ui.LayoutRunner.Companion.bind
 import com.squareup.workflow.ui.ViewBinding
 
-internal class TodoListsLayoutRunner(view: View) : LayoutRunner<TodoListsRendering> {
+internal class TodoListsLayoutRunner(view: View) : LayoutRunner<TodoListsScreen> {
   private val inflater = LayoutInflater.from(view.context)
   private val listsContainer = view.findViewById<ViewGroup>(R.id.todo_lists_container)
 
-  override fun showRendering(rendering: TodoListsRendering) {
+  override fun showRendering(rendering: TodoListsScreen) {
     for ((index, list) in rendering.lists.withIndex()) {
-      addRow(index, list) { rendering.onRowClicked(index) }
+      addRow(
+          index,
+          list,
+          selected = index == rendering.selection && rendering.masterDetailConfig == Master
```

Comment:
Oh crud, I forgot to fix actual display of the selection. It's being set just fine, but nothing visible happens.

Reply:
@zach-klippenstein Changed `selected` to `activated` which seems more correct, but I'm still clueless how to get the style system to acknowledge it. Do you know what the right thing is to do before I just hack something?

Reply:
I think you need to define something in your style to say how to render things in those states?  E.g. a state color drawable for the background of your row widget.